### PR TITLE
LVNETZZZ-47. Create Watchlist endpoint.

### PR DESCRIPTION
**Pull request by taras-tymchuk**
Created watchlist endpoint (GET and POST methods in the controller).
You can find the link to JIRA ticket [here](https://ssu-jira.softserveinc.com/browse/LVNETZZZ-47)

Also added 2 fields for ScheduleDTO class: IsChecked (if a user previously checked this performance as liked) and PerformanceId (to have a possibility to redirect to "Details" page).

### CODE REVIEW

File: TheaterSchedule.DAL/Repositories/ScheduleRepository.cs [hunk 1]
```diff
@@ -13,35 +13,41 @@ public class ScheduleRepository : IScheduleRepository
...
         private TheaterDatabaseContext db;
 
-        public ScheduleRepository(TheaterDatabaseContext context)
+        public ScheduleRepository( TheaterDatabaseContext context )
         {
             this.db = context;
         }
 
-        public IEnumerable<Schedule> GetWithInclude(params Expression<Func<Schedule, object>>[] includeProperties)
+        public IEnumerable<Schedule> GetWithInclude( params Expression<Func<Schedule, object>>[] includeProperties )
```
Comment (vtorkalo) [inline TheaterSchedule.DAL/Repositories/ScheduleRepository.cs]: looks like this method not used - remove it

File: TheaterDatabase/TestData/CleanupDatabase.sql [hunk 2]
```diff
@@ -1,77 +0,0 @@
-USE TheaterDatabase
```
Comment (vtorkalo) [inline TheaterDatabase/TestData/CleanupDatabase.sql]: why this script removed?

File: FillDatabase.bat [hunk 3]
```diff
@@ -0,0 +1,2 @@
+sqlcmd -f 1251 -S DESKTOP-F0OVMC9\SQLEXPRESS -v FullScriptDir="%CD%" -i TheaterDatabase\TestData\FillByTestData.sql
```
Comment (vtorkalo) [inline FillDatabase.bat]: don't commit your connection string

File: TheaterDatabase/TestData/CleanupDatabase.sql (hunk 2 above)
Comment (vladmamai) [reply TheaterDatabase/TestData/CleanupDatabase.sql]: It's okay, I moved cleaning database and filling into one script

**Pull request merged**